examples_gbs/run_tutorial_points.py

```diff
@@ -0,0 +1,109 @@
+"""
+Point Process Tutorial
+======================
+
+This tutorial shows how to generate GBS point process samples and use them to detect outlier
+points in a data set. Point processes are models for generating random point patterns and GBS
+devices can be programmed to operate as special types of point processes that generate clustered
+point patterns. GBS point processes belong to a class of point processes in which the probability
```

Comment:
I'm not sure whether to recommend either giving a reference/hyperlink here or expanding a bit more on the theory.